src/_scss/pages/search/searchSidebar.scss

```diff
@@ -86,6 +86,11 @@
                         height: rem(15);
                         width: rem(15);
                     }
+                    .tooltip-spacer{
```

Comment:
missing space after .tooltip-spacer

Reply:
Fixed, But also #2 design preferred to have the space in between the paragraphs. so kept them the same

Reply:
Ok, please add a comment to the ticket saying this so the expected behavior is clear for QA.

Reply:
She PM me about it and I let her know. BUT I also put it in the ticket just in case